docs/fundamentals/week-2/learning-objectives.md

```diff
@@ -29,10 +29,6 @@ sidebar_label: Learning Objectives
 
 ## PD
 
-- [ ] Explain the purpose of Personal Development
-- [ ] Believe in own capabilities and ability to be successful
-- [ ] Understand own strengths
-- [ ] Identify areas of development
 - [ ] Communicate challenges and barriers in public
 - [ ] Recognise and address fears of transparency
 - [ ] Take active initiative to overcome barriers to success
```

Comment:
```suggestion
- [ ] Take active initiative to overcome barriers to success
```

Still a bit confused by this one. Can you explain more?

Reply:
The PD learning objectives are taken from [the PD competencies rubric](https://docs.google.com/spreadsheets/d/1S0h66lDGXwc1RiuufJDzB5R-YFrr2qNMIq5xxai_hxA/edit#gid=2116060541). I was not involved in their production. I'm trying to match them to the weekly content while moving from the PD website into the Syllabus website.